docs/coordinates/angles.rst

```diff
@@ -100,8 +100,8 @@ Angles will also behave correctly for appropriate arithmetic operations::
     <Angle 3.5 rad>
     >>> np.sin(a / 2)  # doctest: +FLOAT_CMP
     <Quantity 0.479425538604203>
-    >>> a == a
```

Comment:
I couldn't get doctest to think both `( True` and `(True` were the same thing, and I thought this example was sufficiently trivial (and repeated below) that I could just remove it.